for-node-hosts/running-nodes/cronos-mainnet/ksync.md

```diff
@@ -0,0 +1,89 @@
+# KSYNC
+
+This sections covers how to perform a genesis-sync up to live height and how to state-sync to any historical height for Cronos Mainnet with KSYNC. In summary KSYNC is a tool developed by [KYVE](https://www.kyve.network/) which is capable of syncing blocks and state-sync snapshots from the decentralized KYVE data lake directly into Cosmos blockchain nodes. In the case of Cronos all blocks and state-sync snapshots in a 10,000 interval have been validated in a decentralized way and permantenly archived to [Arweave](https://www.arweave.org/), a decentralized permanent storage solution. You can find full documentation on the tool [here](https://docs.kyve.network/validators/ksync).
+
+#### Installation
+
+You can install KSYNC with the following command, ensure that you have go1.21 installed:
+
+```bash
+go install github.com/KYVENetwork/ksync/cmd/ksync@latest
```

Comment:
Hi @troykessler 
1. Installed go: go version go1.22.5 darwin/amd64
2. mkdir - KSYNC
3. ran (zsh:) - KSYNC % go install github.com/KYVENetwork/ksync/cmd/ksync@latest (see image attached)
4. ran ksync version - zsh: command not found: ksync

Repeated the above steps in BASH (see image attached)
<img width="911" alt="Screenshot 2024-07-21 at 18 35 53" src="https://github.com/user-attachments/assets/c5a9b997-c4fc-449b-8352-fc20d55c0f9e">

<img width="1094" alt="Screenshot 2024-07-21 at 18 21 00" src="https://github.com/user-attachments/assets/e087b58d-e861-4318-9d82-d94fb40baa8d">